packages/eas-cli/src/commands/build/index.ts

```diff
@@ -88,6 +90,17 @@ export default class Build extends EasCommand {
       helpValue: 'PROFILE_NAME',
       exclusive: ['auto-submit'],
     }),
+    'build-resource-class': Flags.enum({
+      options: [
+        BuildResourceClass.ANDROID_DEFAULT,
```

Comment:
This is not really consistent with how we name stuff, I would suggest naming it lower case without underscores and platform
"default" | "large" | "m1"